addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -51,6 +54,9 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
      *        the url to load when manually running the tour
      * @param {boolean} [options.rainbowMan=true]
      *        whether or not the rainbowman must be shown at the end of the tour
+     * @param {boolean} [options.sequence]
+     *        priority sequence of the tour (lowest is first, no sequence = comes
+     *        after other sequenced tours in a non deterministic order).
```

Comment:
not sure about this. I would force a default sequence (like 50), so that someone that specify an higher sequence for his tour, ensuring it will be activated *after* the others, even those with no sequence

Reply:
That's what we did for the Systray items, for instance